apps/web/pages/booking/direct/[...link].tsx

```diff
@@ -42,7 +44,7 @@ const pageErrors = {
 const requestSchema = z.object({
   link: z
     .array(z.string())
-    .max(4)
+    .max(4, "Direct link URL must contain exactly 4 parts")
```

Comment:
Should this use t() too perhaps? if we can

Reply:
This is just an error, replacing the not-so-great default zod array.max error "Array must contain at most 4 element(s) accessing"